components/resolved_ts/src/advance.rs

```diff
@@ -106,10 +104,6 @@ impl AdvanceTsWorker {
         let pd_client = self.pd_client.clone();
         let scheduler = self.scheduler.clone();
         let timeout = self.timer.delay(advance_ts_interval);
-        let min_timeout = self.timer.delay(cmp::min(
-            DEFAULT_CHECK_LEADER_TIMEOUT_DURATION,
-            self.advance_ts_interval,
```

Comment:
Why to remove this min_timeout, instead of replacing `self.advance_ts_interval` with the parameter `advanec_ts_interval`?

Reply:
There is already a `timeout` with `advance_ts_interval`, and the `min_timeout` is useless.

Reply:
That timeout could be skipped by `advance_notify`. When notifications are too frequent, `min_timout` could be a rate limiter. That's what I suppose why `min_timeout` exists? If it is removed, is there a possibility of overloading?

Reply:
nice catch, done.